api/v1alpha1/virtualmachine_conversion.go

```diff
@@ -207,19 +207,27 @@ func convert_v1alpha1_VmMetadata_To_v1alpha3_BootstrapSpec(
 }
 
 func convert_v1alpha3_BootstrapSpec_To_v1alpha1_VmMetadata(
-	in *vmopv1.VirtualMachineBootstrapSpec) *VirtualMachineMetadata {
+	btSpec *vmopv1.VirtualMachineBootstrapSpec, annotations map[string]string) *VirtualMachineMetadata {
```

Comment:
Nit: Typical convention is usually `in` & `out` (if present) for the var names in the conversion funcs